bin/IdamHelper.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+const fetch = require('node-fetch');
+const { URLSearchParams } = require('url');
+const btoa = require('btoa');
+
+const USERNAME = 'docassemblytestuser@hmcts.net';
+const PASSWORD = '4590fgvhbfgbDdffm3lk4j';
+const FIRSTNAME = 'docassembly';
+const LASTNAME = 'testuser';
+
+const Env = require('./Env');
```

Comment:
After the changes I suggested above this would look more like:
```
import { idamUrl, oAuthClient, oAuthSecret, oAuthRedirect } from "./Env";
```

Reply:
doesn't seem like I can do with node, even after moving to the latest version, it doesn't seem to support ES6 import statements